config.m4

```diff
@@ -20,7 +20,6 @@ if test "$PHP_XDEBUG" != "no"; then
   CPPFLAGS="$INCLUDES $CPPFLAGS"
 
   AC_CHECK_FUNCS(gettimeofday)
-  AC_CHECK_HEADERS([sys/select.h])
```

Comment:
Hmm, as I understand it poll has been around for a long time so the support should exist on all(?) POSIX systems used today. It was epoll from that other pull request that only has support in Unix. WSAPoll is supported in Windows from Vista/2008 and later. Do you want to have select as a fallback or "wait and see" if there are anyone that needs it?